internal/dependency/manager.go

```diff
@@ -250,22 +250,20 @@ func downloadFile(url string, file io.Writer) (checksum string, err error) {
 
```

Comment:
interesting implementation these three lines:
```
        writer := io.MultiWriter(file, hash)

	if _, err = io.Copy(writer, resp.Body); err != nil {
		return checksum, err
	}
```
Can i change this when i will implement progressbar #7. 

Will be something like this:
```
        writer := io.MultiWriter(file, hash, counter)
```

Reply:
By interesting, you meant not very good? 😅 Of course, you can, but if you want I can fix it right now.

Reply:
You can change it in this PR. In my opinion this is better and much more simple solution.